pages/linux/vgscan.md

```diff
@@ -0,0 +1,13 @@
+# vgscan
+
+> Scan for volume groups on all supported Logical Volume Manager (LVM) block devices.
+> See also `lvm`, `vgchange`.
```

Comment:
I've cross referenced vgchange because together they address the use case:
1. add a disk formatted for LVM to a running system that doesn't get identified by udev
2. run vgscan to determine the name of the volume_group(s) it contains
3. run vgchange to activate one or more volume group so the logical volume can be accessed

Reply:
Is there anything else for me to do here?

Reply:
No, thanks. I just forgot to resolve the conversation.